### PR TITLE
Fixing manger_test.go case.

### PR DESCRIPTION
**Pull request by loriscn12**
Context: https://github.com/golang/go/issues/35472

### CODE REVIEW

File: gnoi/os/manager_test.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func TestSetRunning(t *testing.T) {
 			manager := NewManager("new")
 			err := manager.SetRunning(test.version)
 			if (err == nil) == test.wantsErr {
-				t.Errorf("Error running OS: Error is %w", err)
+				t.Errorf("Error running OS: Error is %v", err)
```
Comment (ericm) [inline gnoi/os/manager_test.go:57]: What version of go did you run into issues with `%w`?

Comment (ericm) [reply gnoi/os/manager_test.go:57]: It was introduced in 1.13

Comment (ericm) [reply gnoi/os/manager_test.go:57]: https://blog.golang.org/go1.13-errors

Comment (loriscn12) [reply gnoi/os/manager_test.go:57]: my bad, I used my personal laptop and did not check the go version, nice you spotted this :)

**Pull request closed without merging**